sys/include/net/gnrc/netif/conf.h

```diff
@@ -54,6 +54,27 @@ extern "C" {
 #define CONFIG_GNRC_NETIF_MSG_QUEUE_SIZE_EXP  (4U)
 #endif
 
+/**
+ * @brief       Packet queue pool size for all network interfaces
+ *
+ * @note        With @ref net_gnrc_sixlowpan_frag the queue should fit at least
+ *              all fragments of the minimum MTU.
+ * @see         net_gnrc_netif_pktq
+ */
+#ifndef GNRC_NETIF_PKTQ_POOL_SIZE
+#define GNRC_NETIF_PKTQ_POOL_SIZE  (16U)
+#endif
+
+/**
+ * @brief       Time in microseconds for when to try send a queued packet at the
+ *              latest
```

Comment:
This is only a fallback in case the driver doesn't send a notification for TX/RX done, right?

Reply:
Or if it is missed for some reason, yes. Should I clarify that? Should I make the timer optional maybe?

Reply:
Yes a clarification that this should only be a fallback would be good.
Or if the hardware does not support TX done reporting (does such hardware exist?)

Reply:
Done. See below.